resource-aggregate/config.yaml

```diff
@@ -0,0 +1,85 @@
+log:
+  debug: false
+apis:
+  grpc:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+    authorization:
+      authority:
+      audience:
+      ownerClaim: sub
+      http:
+        maxIdleConns: 16
+        maxConnsPerHost: 32
+        maxIdleConnsPerHost: 16
+        idleConnTimeout: 30s
+        timeout: 10s
+        tls:
+          caPool: "/secrets/public/rootca.crt"
+          keyFile: "/secrets/private/cert.key"
+          certFile: "/secrets/public/cert.crt"
+          useSystemCAPool: false
+clients:
+  eventBus:
+    nats:
+      url:
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+  eventStore:
+    mongoDB:
+      uri:
+      database: eventStore
+      tls:
+        caPool: "/secrets/public/rootca.crt"
+        keyFile: "/secrets/private/cert.key"
+        certFile: "/secrets/public/cert.crt"
+        useSystemCAPool: false
+      # limits number of queries in one find request.
```

Comment:
what is one find request?

Reply:
When you get events (find function in mongo) from eventstore it can goes through parallel request.  And this it a limit how many query parameters will be in one get events(find for mongo).